src/plots/linePlot.ts

```diff
@@ -14,6 +14,9 @@ export module Plots {
 
     private _autorangeSmooth = false;
 
+    // Performance options
```

Comment:
There will be (with downsampling). It's easier to not have a merge conflict / forget to add it if I do this.